qa-tests-backend/src/test/groovy/NetworkBaselineTest.groovy

```diff
@@ -193,5 +235,102 @@ class NetworkBaselineTest extends BaseSpecification {
         )
         validateBaseline(deferredBaselinedClientBaseline, beforeDeferredCreate, justAfterDeferredCreate,
             [new Tuple2<String, Boolean>(serverDeploymentID, false)])
+
+        when:
+        "Create another deployment, ensure it DOES NOT get added to serverDeploymentID due to user lock"
+        NetworkBaselineService.lockNetworkBaseline(serverDeploymentID)
+
+        batchCreate([DEFERRED_POST_LOCK_CLIENT_DEP])
+
+        def postLockClientDeploymentID = DEFERRED_POST_LOCK_CLIENT_DEP.deploymentUid
+        assert postLockClientDeploymentID != null
+        println "Post Lock Deployment: ${postLockClientDeploymentID}"
+
+        // Waiting on it to come out of observation.
+        def postLockClientBaseline = evaluateWithRetry(30, 4) {
+            def baseline = NetworkBaselineService.getNetworkBaseline(postLockClientDeploymentID)
+            def now = System.currentTimeSeconds()
+            if (baseline.getObservationPeriodEnd().getSeconds() > now) {
+                throw new RuntimeException(
+                    "Baseline ${postLockClientDeploymentID} is not out of observation yet. Baseline is ${baseline}"
+                )
+            }
+            return baseline
+        }
+        assert postLockClientBaseline
+
+        assert NetworkGraphUtil.checkForEdge(postLockClientDeploymentID, serverDeploymentID, null, 180)
+        serverBaseline = evaluateWithRetry(30, 4) {
+            def baseline = NetworkBaselineService.getNetworkBaseline(serverDeploymentID)
+            if (baseline.getPeersCount() > 2) {
```

Comment:
The way this test works, is at this point in time we already have 2 peers baselined AND we have locked the server deployment baseline.  So when we add the post lock client deployment it should not get added as a peer to the server deployment.  So we check the server deployment baseline and if we add another peer (when the client deployment comes out of observation) then we will fail the test.  Let me think about it a minute because there might be a better way to do this.  In reality all that step is doing is buying time for the post lock client deployment to come out of observation and process its flows.  The real validation would occur further down with the statement:

`validateBaseline(postLockClientBaseline, beforeDeferredCreate, justAfterDeferredCreate,
            [])`

So I'm certain I can improve this and have it make more sense.